hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3ClientFactory.java

```diff
@@ -53,25 +52,14 @@
  */
 @InterfaceAudience.LimitedPrivate("HBoss")
 @InterfaceStability.Evolving
-@Deprecated
 public interface S3ClientFactory {
 
-  /**
-   * Creates a new {@link AmazonS3} client.
-   *
-   * @param uri S3A file system URI
-   * @param parameters parameter object
-   * @return S3 client
-   * @throws IOException IO problem
-   */
-  AmazonS3 createS3Client(URI uri,
-      S3ClientCreationParameters parameters) throws IOException;
 
   /**
    * Creates a new {@link S3Client}.
```

Comment:
Shall we now rename this to `createS3Client`?

Reply:
I'm adding this method back in now, as removing it completely is slightly more complicated. I'll do that in a seperate PR.